agents/loco_mc_agent.py

```diff
@@ -366,6 +366,8 @@ def register_hook(self, receiver, sender):
         """
         if sender == self.perceive:
             self._dispatch_signal.connect(receiver, sender)
+        else:
+            raise ValueError("{} is not allowed to register hooks".format(sender.__name__))
```

Comment:
in the error message, please give the options available if this particular `sender` is not allowed.

Ideallly, you'd restructure this to:

```
allowed  = [self.perceive,]
if sender in allowed:
    self._dispatch_signal.connect(receiver, sender)
else:
    raise ValueError("Unknown hook event {}. Available options are: {}".format(sender.__name__, allowed))
```

Reply:
In your opinion, would it be more readable to have `allowed = [self.perceive.__name__,]`? Otherwise, the error message is a bit longer `Available options are: [<function xyz at some memory address>]`

Reply:
yea, sorry that's what I meant.

Reply:
Got it, I've added these changes to [74e6060](https://github.com/facebookresearch/droidlet/pull/500/commits/74e6060ed1d1482ef9fe44eee19e7164a48771d5) :)